src/ui/components/App.tsx

```diff
@@ -107,8 +107,14 @@ function App({ children, modal, theme }: AppProps) {
     // Stop space bar from being used as a universal "scroll down" operator
     // We have a big play/pause interface, so space makes way more sense for that.
     window.addEventListener("keydown", function (e) {
-      if (e.code === "Space" && e.target === document.body) {
-        e.preventDefault();
+      if (e.code === "Space") {
+        if (
+          e.target === document.body ||
+          (e.target?.hasOwnProperty("classList") &&
+            (e.target as Element).classList.contains(".CodeMirror-scroll"))
+        ) {
+          e.preventDefault();
+        }
       }
     });
   });
```

Comment:
Needs a dependencies array and a return value as a function that cleans up the event listener from `window` (not that it probably makes literally any difference, but it's a good practice).